README.md

```diff
@@ -4,12 +4,44 @@
 
 #### School Year
 
-### 2020 - 2021
+### 2021 - 2022
 
 - **Ying Yang** - Product Manager
 - **Jessica Feng** - Technical Project Manager
+- **Sarah Mirae Kim** - Associate Product Manager
 - **Sheri Guo** - Product Designer
 - **Emma Shen** - Product Designer
 - **Katrina Wang** - Software Developer
 - **Rohan Maheshwari** - Software Developer
 - **Zak Kent** - Software Developer
+- **Sydney Tran** - Software Developer
+- **Sarah Young** - Software Developer
+- **Vishok Jonnalagadda** - Software Developer 
+
+### 2020 - 2021
+
+- **Yvonne Chan** - Product Manager
+- **Andrew Gao** - Product Manager 
+- **Kaushik Ravikumar** - Technical Project Manager 
+- **Ansh Godha** - iOS Developer 
+- **Mihikaa Goenka** - iOS Developer 
+- **Changyuan Lin** - iOS Developer 
+- **Toby Ko** - Android Developer 
+- **Jessica Feng** - Backend Developer 
+- **Vaishnavi Gupta** - Backend Developer 
+- **Buma Gana** - Designer
+- **Rebecca Fu** - Designer 
+- **Isabella Zhou** - Designer 
+
+### 2019 - 2020
+
+- **Yvonne Chan** - Product Manager 
+- **Asneel Das** - Technical Project Manager 
+- **Ansh Godha** - iOS Developer 
+- **Mihikaa Goenka** - iOS Developer 
+- **Changyuan Lin** - iOS Developer 
+- **Kaushik Ravikumar** - Android Developer 
+- **Andrew Gao** - Android Developer 
+- **Benjamin Shen** - Backend Developer 
+- **Kaitlyn Son** - Designer 
```

Comment:
These people are actually a part of a different team, so just keep the ReadME that we had before and add on the 2021-2022 people (so that's everyone from the list you currently have except Sheri)